src/pages/OfferNew/Checkout/InsuranceSummary.tsx

```diff
@@ -55,31 +54,34 @@ const Value = styled('div')`
 `
 
 interface Props {
-  firstQuote: CompleteQuote
+  offerData: OfferData
+  ssn?: string
 }
 
-export const InsuranceSummary: React.FC<Props> = ({ firstQuote }) => {
+export const InsuranceSummary: React.FC<Props> = ({ offerData, ssn }) => {
   const textKeys = useTextKeys()
 
   return (
     <Wrapper>
       <Title>{textKeys.CHECKOUT_SUMMARY_TITLE()}</Title>
-      <Table>
-        {getDetails(
-          firstQuote.quoteDetails,
-          textKeys,
-          firstQuote.ssn ?? undefined,
-        ).map((group, index) => (
-          <Group key={index}>
-            {group.map(({ key, value, label }) => (
-              <Row key={key}>
-                <Label>{label}</Label>
-                <Value>{value}</Value>
-              </Row>
-            ))}
-          </Group>
-        ))}
-      </Table>
+      {offerData.quotes.map((quote) => {
+        return (
+          <Table>
```

Comment:
Could/should add a key here, however I more or less only do this to "break the design" so we can fix it